docker/wrapper.py

```diff
@@ -56,8 +56,25 @@ def call_pipeline(mount, args):
         else:
             log.info('Flag "--no-clean" was used, therefore {} was not deleted.'.format(work_dir))
 
+    log.info("output dir is {} and files are:\n{}:".format(mount,'\n'.join(os.listdir(mount))))
+    fail_files = [x for x in os.listdir(mount) if x.startswith('FAIL.')]
+    log.info("fail files are:\n{}:".format('\n'.join(fail_files)))
+    for fail_file in fail_files:
+        new_file = fail_file.lstrip('FAIL.')
```

Comment:
How about new_file = fail_file[len('FAIL.'):]
so no one misses it if they change 'FAIL' above the line?